glsl/src/parsers.rs

```diff
@@ -1123,7 +1123,7 @@ pub fn cond_expr(i: &str) -> ParserResult<syntax::Expr> {
       cut(terminated(char(':'), blank)),
       cut(assignment_expr),
     )),
-    a,
+    move || a.clone(),
```

Comment:
Argh it’s a pity it’s not linear in its argument and that it forces to clone for the initial argument.